lib/std/math/big/int.zig

```diff
@@ -424,7 +433,10 @@ pub const Mutable = struct {
     ///
     /// Asserts r has enough elements to hold the result. The upper bound is `@max(a.limbs.len, b.limbs.len)`.
     fn addCarry(r: *Mutable, a: Const, b: Const) bool {
-        if (a.eqlZero()) {
+        if (a.eqlZero() and b.eqlZero()) {
```

Comment:
Either `copy` should produce a normalized value, or this should be moved into the next case as just `if (b.eqlZero())` to avoid calling `a.eqlZero` twice in a row.